tests/test_utils.py

```diff
@@ -8,10 +8,10 @@
 class UtilsGetLocalFileContentTestCase(unittest.TestCase):
     def test_unicode_file_content(self):
         content = utils.get_local_file_content("tests/content_unicode.txt")
-        self.assertTrue(isinstance(content, unicode))
+        self.assertTrue(isinstance(content, str))
```

Comment:
Oh, actually, now that I look at the full test, I think the `assertIn()` checks already have us covered on the file parsing and string encoding—as long as it can correctly read in and save the non-ASCII character, we can just remove these two `assertTrue()` checks.